Content.Server/Blocking/BlockingSystem.cs

```diff
@@ -0,0 +1,72 @@
+using Content.Server.Examine;
+using Content.Shared.Blocking;
+using Content.Shared.Damage;
+using Content.Shared.Damage.Prototypes;
+using Content.Shared.Verbs;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Blocking;
+
+public sealed class BlockingSystem : SharedBlockingSystem
+{
+    [Dependency] private readonly ExamineSystem _examine = default!;
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<BlockingComponent, GetVerbsEvent<ExamineVerb>>(OnVerbExamine);
+    }
+
+    private void OnVerbExamine(EntityUid uid, BlockingComponent component, GetVerbsEvent<ExamineVerb> args)
+    {
+        if (!args.CanInteract || !args.CanAccess)
+            return;
+
+        _proto.TryIndex<DamageModifierSetPrototype>(component.PassiveBlockDamageModifer, out var passiveblockModifier);
+        _proto.TryIndex<DamageModifierSetPrototype>(component.ActiveBlockDamageModifier, out var activeBlockModifier);
+
+        var fraction = component.IsBlocking ? component.ActiveBlockFraction : component.PassiveBlockFraction;
+        var modifier = component.IsBlocking ? activeBlockModifier : passiveblockModifier;
```

Comment:
You should decide which of these prototypes you need prior to calling `_proto.TryIndex` to avoid doing two lookups when you only want one.